docs/trulens_explain/attribution_parameterization.md

```diff
@@ -60,30 +62,32 @@ Now suppose you want to know the attribution of some internal layer on the final
 **Use cases:** 
 
 * As a preprocessing step, you drop a feature, so do not need attributions on that.
-* For torch models, model inputs are not tensors. so you'd want the 'in' anchor of the first layer.  
+* For PyTorch models, model inputs are not tensors, so you'd want the 'in' anchor of the first layer.  
 
 **Cuts needed:** As you want to know the affect of some other layer rather than the input layer, you need to customize the attribution cut.  
 **Model inputs** → InputCut  
 **Attribution Cut** → Your attribution layer (The layer you want to assign importance/attributions with respect to output), anchor:'in'  
 **QoI Cut** → OutputCut
 
 ### Advanced Use Cases
-For the following use cases, it may help to see the [Advanced Definitions](#advanced-definitions)
+For the following use cases, it may help to refer to [Advanced Definitions](#advanced-definitions).
 
 #### Case 4: The Distribution of Interest (DoI) Cut / Explanation flexibility
 
-Usually, we explain the output with respect to each point in the input. All cases up to now were using a default called PointDoI. Now, suppose you want to explain using an aggregate over samples of points.  
+Usually, we explain the output with respect to each point in the input. All cases up to now were using a default called `PointDoI`. Now, suppose you want to explain using an aggregate over samples of points.  
+
+**Use case:** You want to perform approaches like Integrated Gradients, Grad-CAM, Shapley values instead of saliency maps. These only differ by sampling strategies.
+
+> E.g. Integrated Gradients is a sample from a straight line from a baseline to a value.
 
-**Use case:** You want to do integrated gradients, gradcam, shapley values instead of saliency maps. These only differ by sampling strategies.  
-Eg: Integrated gradients is a sample from a straight line from a baseline to a value.  
 **Cuts needed:** Define a DoI that samples from the default attribution cut.  
 **Model inputs** → InputCut  
 **DoI/Attribution Cut** → Your baseline/DoI/attribution layer, anchor:'in'  
 **QoI Cut** → OutputCut
  
 #### Case 5: Internal explanations
 
-**Use case:** You want to explain an internal layer. Things like integrated gradients are a DoI on the baseline to the value, but it is on the layer the baseline is defined.
+**Use case:** You want to explain an internal layer. Methods like Integrated Gradients are a DoI on the baseline to the value, but it is located on the layer the baseline is defined.
```

Comment:
Note: the wording here was initially confusing, looking for feedback on how to improve readability here.